env.sample

```diff
@@ -0,0 +1,10 @@
+BATCH_JOB_QUEUE_NAME=
+GCC_ACCOUNT_ID=
```

Comment:
The `GCC` prefixes are the old acronym representing what is now MCP, so we should use `MCP` as the prefix.

Reply:
Thanks! I don't even know what `GCC` means 🙈 I'll update this